docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/jenkins-troubleshooting.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Jenkins Troubleshooting
+description: This article describes how to troubleshoot the most common problems with the jenkins scheulder
+template: troubleshooting-guide-template
+---
+
+<!-- Locate your guide to one of the troubleshooting sections:
+
+* Troubleshooting installation issues
+* Troubleshooting Spryker in Docker issues
+* Troubleshooting general technical issues
+
+Every troubleshooting article should have its own page in the respective section.-->
+
+## Title
+
+Jenkins does not restart properly due to long running job
```

Comment:
Why does Jenkins need to restart during a deployment? How does Jenkins not being able to restart fails a deployment?

Reply:
to apply changes made during deployment